src/main/java/org/hmhb/oauth/GoogleResponseData.java

```diff
@@ -0,0 +1,60 @@
+package org.hmhb.oauth;
+
+import javax.annotation.Nonnull;
+import javax.annotation.concurrent.Immutable;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleTokenResponse;
+import com.google.api.services.plus.model.Person;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * An object to hold the user's information from google's responses.
+ */
+@Immutable
+public class GoogleResponseData {
+
+    private final GoogleTokenResponse googleOauthToken;
```

Comment:
I created this object so the service could return what it used to, and also the g+ profile info.